pywwise/ak/wwise/core/profiler.py

```diff
@@ -21,88 +35,503 @@ def __init__(self, client: _WaapiClient):
 		self.state_changed: _RefEvent
 		self.switch_changed: _RefEvent
 	
-	def enable_profiler_data(self):
+	def enable_profiler_data(self, data_types: set[tuple[EDataTypes, bool]] = None) -> bool:
 		"""
+		https://www.audiokinetic.com/en/library/edge/?source=SDK&id=ak_wwise_core_profiler_enableprofilerdata.html \n
 		Specifies the type of data you want to capture. Overrides the user's profiler settings.
+		:param data_types: Tuple that contains the data type you want to capture via an enum and if profiler capture
+		will be enabled for said data type (Defaults to true).
+		:return: It returns a bool indicating whether the call was successful or not.
 		"""
+		if data_types is None:
+			return False
+		
+		args = {"dataTypes": list()}
+		
+		for data_type in data_types:
+			args["dataTypes"].append({"dataType": data_type[0], "enabled": data_type[1]})
+		
+		return self._client.call("ak.wwise.core.profiler.enableProfilerData", args) is not None
 	
-	def get_audio_objects(self):
-		"""
-		Retrieves the Audio Objects at a specific profiler capture time.
-		"""
+	def get_audio_objects(self, time: ETimeCursor | int, bus_pipeline_id: int = None,
+	                      return_options: set[EAudioObjectOptions] = None) -> tuple[AudioObjectInfo, ...]:
+		"""
+		https://www.audiokinetic.com/en/library/edge/?source=SDK&id=ak_wwise_core_profiler_getaudioobjects.html \n
+		Retrieves the Audio Objects at a specific profiler capture time. \n
+		:param time: Time in milliseconds to query for Audio Objects, or a Time Cursor from which to acquire the time.
+					 This parameter can have 2 possible values: int or ETimeCursor. The int is the time to query.
+					 The ETimeCursor can have two values: user or capture. The User Time Cursor is the one that can
+					 be manipulated by the user, while the Capture Time Cursor represents the latest time of the current
+					 capture.
+		:param bus_pipeline_id: Unsigned Integer 32-bit. Range: [0,4294967295] The pipeline ID of a Bus instance to get.
+		:param return_options: Members to return for each Audio Object. Defaults to Audio Object ID, Bus Pipeline ID,
+							   Instigator Pipeline ID and Effect Class ID.
+		:return: For each profiled audio object, an AudioObjectInfo containing an object's ID, pipeline ID,
+				 instigator pipeline ID, effect class ID and a dictionary containing additional information
+				 (requested via `return_options`). When accessing the values in the dictionary, use the
+				 EAudioObjectOptions enum as the keys. If this function call fails, an empty tuple is returned.
```

Comment:
GitHub's diff **_might_** be trolling... Check the docstring alignment here, just in case!